v2/trace/oc/b3_dd_propagator.go

```diff
@@ -0,0 +1,151 @@
+// Package oc contains a propagation.HTTPFormat implementation for B3 and Datadog propagation.
```

Comment:
Missing doc.go

Reply:
missing the file or some content?

Reply:
You might be able to reuse some of the content, but the file is missing and would like to have an small example of usage :)

Reply:
yep, will add an example